config.properties.template

```diff
@@ -25,6 +25,9 @@ random_next_pokestop_selection=5
 # Meters per second
 speed=2.8
 
+# Should the bot follow the streets (true) or just go directly to pokestops/waypoints
+follow_streets = true
```

Comment:
and maybe make the default false, some people use this bot for max xp/hour; I guess this doesn't result in more xp/hour